lua/gt.lua

```diff
@@ -16,8 +16,13 @@ return function (net_conf, numa_table)
 		n_lcores)
 	gatekeeper.gt_assign_lcores(gt_conf, gt_lcores)
 
+	local net_prefix = "10.0.2.1/24"
+	local net_prefix6 = "2003:db8::1/32"
+	gt_conf.max_num_ipv6_neighbors = 65535
```

Comment:
Since this is going to be the size of the hash table, are there any heuristics we should use to make this good for hashing? For example, is this purposely one less than a power of 2? If so, maybe we should leave a comment to that effect.